### PR TITLE
Bring Python current in Travis

### PR DESCRIPTION
**Pull request by gliptak**
https://travis-ci.org/capitalone/Data-Load-and-Copy-using-Python/builds/546780081

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 language: python
-dist: trusty
+dist: xenial
 sudo: false
 cache:
   pip: true
 
 python:
-  - "3.5"
```
Comment (fdosani) [inline .travis.yml]: Is it possible to retain 3.5 for testing?

Comment (gliptak) [reply .travis.yml]: updated

**Comment by fdosani** [comment]
Hi @gliptak Sorry for the bother, I'm wondering if you can sign the CLA located here please: https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform

**Comment by gliptak** [comment]
@fdosani signed

**Pull request merged**